src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/HubProtocolExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
+{
+    public static class HubProtocolExtensions
```

Comment:
Should this be public?

Reply:
It's either that or `InternalsVisibleTo`. Which we generally avoid for product-to-product references. It's used in a bunch of places. It's in an `Internal` namespace.